LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2015 The cargo-readme Developers
```

Comment:
Was this left in by accident? The cargo-readme devs definitely don’t have copyright over this code 😅

Reply:
Wow my bad, just copied the first MIT license I saw. Something I should put in place of that line or just remove it?